charts/kashti/templates/deployment.yaml

```diff
@@ -20,8 +20,8 @@ spec:
     spec:
       containers:
         - name: {{ .Chart.Name }}
-          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
-          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          image: "{{ .Values.image.repository }}:{{ default .Chart.AppVersion .Values.image.tag }}"
+          imagePullPolicy: {{ default "IfNotPresent" .Values.image.pullPolicy }}
```

Comment:
The other charts in this repo already use this convention-- get app version from `Charts.yaml` unless _overridden_ in `values.yaml`. This prevents the version/tag from having to be updated in two separate places on each release of the chart.